e2e/e2e-utils/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@tanstack/router-e2e-utils",
+  "private": true,
+  "scripts": {
+    "clean": "rimraf ./dist && rimraf ./coverage",
+    "test:eslint": "eslint ./src",
+    "test:unit": "exit 0; vitest --typecheck",
+    "build": "vite build && tsc --noEmit"
+  },
+  "type": "module",
+  "types": "dist/esm/index.d.ts",
+  "main": "dist/cjs/index.cjs",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/esm/index.d.ts",
+        "default": "./dist/esm/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/index.d.cts",
+        "default": "./dist/cjs/index.cjs"
+      }
+    },
+    "./package.json": "./package.json"
+  },
```

Comment:
I think we should structure this package a bit differently: There isn’t an actual need to build this, as we are never publishing this. So we could have `exports` point to `src` directly:

- https://github.com/TanStack/router/pull/3301